Test/QuickCheck/Arbitrary.hs

```diff
@@ -166,26 +153,12 @@ import qualified Data.IntMap as IntMap
 import qualified Data.Sequence as Sequence
 
 import qualified Data.Monoid as Monoid
-import Data.Array.IArray
-import Data.Array.Unboxed
 
 #ifndef NO_TRANSFORMERS
 import Data.Functor.Identity
 import Data.Functor.Constant
 import Data.Functor.Compose
 import Data.Functor.Product
-
-#ifdef MIN_VERSION_transformers
-#if MIN_VERSION_transformers(0,4,0)
-#define TRANSFORMERS_SUM
-#endif
-#else
-#define TRANSFORMERS_SUM
-#endif
-
-#ifdef TRANSFORMERS_SUM
-import Data.Functor.Sum
-#endif
```

Comment:
but it is in `transformers-compat`, and actually migrated to `base` recently.

So I went with simple and straight-forward no-CPP approach, as with `Natural` and `NonEmpty`.